SQL/New_patches/2019-05-03-deprecate_useprojects.sql

```diff
@@ -2,8 +2,8 @@ DELETE cs, c FROM ConfigSettings cs JOIN Config c ON c.ConfigID=cs.ID WHERE cs.N
 
 -- if Project table is empty, add default 'loris' project
 INSERT INTO Project (Name)
-SELECT 'loris'
-WHERE NOT EXISTS (SELECT * FROM Project);
+(SELECT 'loris' FROM DUAL
```

Comment:
@PapillonMcGill what is DUAL ?

Reply:
it's a fake table with exactly one row (it originates from Oracle, but the SQL standard doesn't really have any useable equivalent for cases like this and MySQL and MariaDB seem to both support it)

Reply:
lol... learning stuff everyday